app/components/clinic/PatientForm.js

```diff
@@ -329,6 +331,10 @@ export const PatientForm = (props) => {
     dispatch(actions.async.sendPatientDexcomConnectRequest(api, selectedClinicId, patient.id));
   }
 
+  function fetchPatientDetails() {
+    dispatch(actions.async.fetchPatientFromClinic.bind(null, api, selectedClinicId, patient.id)());
```

Comment:
Is there an advantage to using the `.bind()` here if the function is always immediately executed?

Reply:
Ah, good catch.  Will forego the bind.